package.json

```diff
@@ -7,10 +7,14 @@
     "yarn": "1.5.1"
   },
   "scripts": {
-    "heroku-postbuild": "sh scripts/heroku-postbuild.sh",
-    "heroku-serve": "http-server"
+    "clean": "rm -rf node_modules && rm package-lock.json",
+    "postinstall": "cd node_modules/vets-website && yarn install --production=false",
+    "build": "INSTALL_HOOKS=no npm --prefix node_modules/vets-website run build -- --content-deployment --content-directory=${PWD}/pages --entry static-pages --brand-consolidation-enabled",
+    "rebuild": "npm run clean && npm install && npm run build",
+    "start": "npm --prefix node_modules/vets-website run heroku-serve build/localhost",
+    "heroku-postbuild": "npm run build && npm run clean"
   },
   "dependencies": {
-    "http-server": "^0.9.0"
+    "vets-website": "github:department-of-veterans-affairs/vets-website.git#content-deployment-basic"
```

Comment:
This is all pretty funky looking, but it's ultimately installing `vets-website` as a module to use its build. I have it pointed to the  `content-deployment-basic` branch while I developed, but it will be master once `vets-website` is ready.

The main piece here is on line 12, which builds the site with that `--content-deployment` flag. The Jenkins side of things will need to do the same.

Reply:
We run clean in `heroku-postbuild`, because otherwise the archive size was too large for Heroku to deploy it. By cleaning it, we solve that problem, but we lose all of our caching. 

This ticket, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14694, addresses that issue by installing only the modules the content deploy needs as regular dependencies and the rest installed as devDependencies.